### PR TITLE
chore: Update generator with knobs import; add events to templates

### PR DESCRIPTION
**Pull request by castastrophe**
## Infrastructure updates

Updating the generator to support events, readme injection, and fixing the reference to the add-on knobs import which no longer requires the `polymer` classification.

### Testing instructions

- [x] `npm run new`
  - Be sure to add a few events
  - Check the compiled storybook result renders with no errors

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Repository compiles and tests pass.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
         "../elements/pfe-cta/test/index.html",
         "../elements/pfe-datetime/test/index.html",
         "../elements/pfe-dropdown/test/index.html",
+        "../elements/pfe-foo/test/index.html",
```
Comment (castastrophe) [inline test/index.html:26]:
```suggestion
```

File: generators/element/templates/demo/element.story.ejs [hunk 2]
```diff
@@ -17,14 +25,28 @@ const defaultContent = tools.autoContent(1, 2);
 
 stories.addDecorator(bridge.withKnobs);
 
+<% if (events.length > 0) { -%>
+// Log events
+<% for(let i = 0; i < events.length; i++) { -%>
+stories.addDecorator(withActions(`${<%= elementClassName %>.tag}:<%= events[i] %>`));
```
Comment (zhawkins) [inline generators/element/templates/demo/element.story.ejs:31]: I'm seeing an `withActions` is not defined error. I don't see it being imported either. 🤔

Comment (zhawkins) [reply generators/element/templates/demo/element.story.ejs:31]: If it helps: 
image

**Review by zhawkins — CHANGES_REQUESTED** [review]
Got stuck on a storybook error. Full disclosure here's what I did to get here:

1. `npm install`
1. `npm run new` (added a new element with two events.)
1. `npm run build`
1. `npm run build-storybook`
1. `npm run storybook`

File: generators/element/templates/demo/element.story.ejs (hunk 2 above)
Comment (castastrophe) [reply generators/element/templates/demo/element.story.ejs:31]: You're right! `import { withActions } from "@storybook/addon-actions";`

File: generators/element/templates/demo/element.story.ejs [hunk 3]
```diff
@@ -1,11 +1,19 @@
 import { storiesOf } from "@storybook/polymer";
-import * as bridge from "@storybook/addon-knobs/polymer";
+import * as bridge from "@storybook/addon-knobs";
```
Comment (castastrophe) [inline generators/element/templates/demo/element.story.ejs:2]:
```suggestion
import * as bridge from "@storybook/addon-knobs";
import { withActions } from "@storybook/addon-actions";
```

File: generators/element/templates/demo/element.story.ejs (hunk 2 above)
Comment (castastrophe) [reply generators/element/templates/demo/element.story.ejs:32]: Added it!

**Review by zhawkins — APPROVED** [review]
Looks good!

**Pull request merged**